pkg/identity/numericidentity.go

```diff
@@ -29,6 +29,10 @@ const (
 	// shifted
 	ClusterIDShift = 16
 
+	// LocalIdentityFlag is the bit in the numeric identity that identifies
+	// a numeric identity to have local scope
+	LocalIdentityFlag = NumericIdentity(1 << 24)
+
```

Comment:
I can't comment on it directly, but the comment on  `type NumericIdentity` also needs an (line 288)